simpleviewinc/keg-components
# Keg cli updates

**Ticket**: [ZEN-145](https://jira.simpleviewtools.com/browse/ZEN-145)

## Context

* With the recent update to using packages, the build folder files are typically not used
  * This is why the post install script runs `git update assume-unchange ./build`
* When working locally, you'll get an error; because the import for `re-theme` does not exist
  * This will only happen, if you use the currently built files
  * If you run the build command, the build folder should be updated
  * We should not have to run the build command, for keg-components to work
  * `re-theme` should be imported from the `@simpleview` folder

## Goal

* Make keg-components work properly without needing to run the build command

## Updates

* Fixed `re-theme` imports the build folder files
* Added `.dockerignore` file

## Testing

* Look at the package.json
  * See `@simpleviewinc/re-theme` in `devDependencies`
* Look at the current files in the build folder
  * See they are importing `re-theme` without the `@simpleviewinc/`
